### PR TITLE
[FLINK-18394][docs-zh] Translate "Parquet Format" page into Chinese

### PR DESCRIPTION
**Pull request by Luxios22**
## What is the purpose of the change

Translate "Parquet Format" page into Chinese.

The page url is https://ci.apache.org/projects/flink/flink-docs-master/zh/dev/table/connectors/formats/parquet.html

The markdown file is located in flink/docs/dev/table/connectors/formats/parquet.zh.md

## Brief change log

- Translate `flink/docs/dev/table/connectors/formats/parquet.zh.md`.

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

File: docs/dev/table/connectors/formats/parquet.zh.md [hunk 1]
```diff
@@ -186,4 +185,4 @@ The following table lists the type mapping from Flink type to Parquet type.
     </tbody>
 </table>
 
-<span class="label label-danger">Attention</span> Composite data type: Array, Map and Row are not supported.
+<span class="label label-danger">注意</span> 复合数据类型（Array、Map 与 Row）不被支持。
```
Comment (JingsongLi) [inline docs/dev/table/connectors/formats/parquet.zh.md:188]: 暂不支持复合数据类型（Array、Map 与 Row）

Comment (Luxios22) [reply docs/dev/table/connectors/formats/parquet.zh.md]: Thanks for your review, I've submitted the requested change.

**Comment by Luxios22** [comment]
Thanks for your review, I've submitted the requested change.

**Comment by JingsongLi** [comment]
> Thanks @Luxios22 for contribution, parquet format can be consistent with json Chinese page.

Hi @Luxios22 , I mean the Chinese words can be consistent with JSON Chinese page. Like "是否必须" "默认值". And some sentence. Actually the structure in parquet has many repeated words to JSON.

**Comment by Luxios22** [comment]
I modified my words according to the JSON page this time.

**Comment by JingsongLi** [comment]
Sorry for the late merging, merging...

**Pull request merged**